siliconcompiler/schema/schema_cfg.py

```diff
@@ -188,14 +199,90 @@ def schema_cfg():
     # Modeling
     cfg = schema_model(cfg)
 
-    # Datasheeet
+    # Schematic Entry
+    cfg = schema_schematic(cfg)
+
+    # Datasheet
     cfg = schema_datasheet(cfg)
 
     # Packaging
     cfg = schema_package(cfg)
 
     return cfg
 
+###############################################################################
+# Schematic
+###############################################################################
+
+def schema_schematic(cfg):
+    ''' Schematic configuration
+    '''
+
+    name = 'default'
+    filetype = 'default'
+
+    scparam(cfg,['schematic', 'component', name],
+            sctype='str',
+            shorthelp="Schematic: component (instance)",
+            switch="-schematic_component 'name <str>'",
+            example=[
+                "cli: -schematic_component 'i0 za001'",
+                "api:  chip.set('schematic', 'component', 'i0', 'za001')"],
+            schelp="""Unique manufacturer part number (MPN) of the  named
+            component.""")
+
+    scparam(cfg, ['schematic', 'net', name],
```

Comment:
+1 to Noah's comment that we might want to discuss some of the `'schematic'` values a bit more. I like the idea of letting designers tell SC where core components like macros and pins should be placed, but net names might be closer to the "duplicating details which are represented in source files" end of the spectrum.